### PR TITLE
creates schedule task and restart after install

### PR DESCRIPTION
**Pull request by ubozov**
Resolves BV-1202.

### CODE REVIEW

File: inst/command/funcs.go [hunk 1]
```diff
@@ -264,6 +265,23 @@ func startServices(o *openvpn.OpenVPN) error {
...
+	if !o.IsWindows || !strings.EqualFold(o.Role, "server") {
+		return nil
+	}
+
+	time.Sleep(10 * time.Second)
+	if err := stopService(o); err != nil {
+		return fmt.Errorf("failed to stop services: %v", err)
+	}
+
+	time.Sleep(10 * time.Second)
```
Comment (dzeckelev) [inline inst/command/funcs.go]: Why wait 20 seconds? Can this be optimized?

Comment (ubozov) [reply inst/command/funcs.go]: the first delay in 10 sec is needed to wait start the previous launch of the service
the second delay is reductant - deleted

Comment (dzeckelev) [reply inst/command/funcs.go]: Why not check that the service really stopped before starting? I understand why we need timeouts, but this is not effective.

Comment (ubozov) [reply inst/command/funcs.go]: updated

**Pull request merged**